skiko/src/androidMain/kotlin/org/jetbrains/skiko/PlatformInput.android.kt

```diff
@@ -0,0 +1,5 @@
+package org.jetbrains.skiko
+
+actual interface PlatformInput {
```

Comment:
Is there an actual `interface PlatformInput`? I can't find it in this branch.

Reply:
I guess PlatformInput was replaced by SkikoInput?

Reply:
Yeah, you right!
Fixed